web/template/admin/course_settings.gohtml

```diff
@@ -65,6 +65,15 @@
                         {{if not .}}checked{{end}}>
                 <span class="ml-2">Allow Anonymous Messages</span>
             </label>
+            <label class="inline-flex items-center pl-2" for="enChatMod">
+                <input :disabled="!enChat"
+                       class="w-auto form-checkbox"
+                       id="enChatMod" name="enChatMod"
+                       type="checkbox"
+                       {{if and . .ModeratedChatEnabled}}checked{{end}}
+                        {{if not .}}checked{{end}}>
+                <span class="ml-2">Activate Moderate Chat</span>
```

Comment:
For consistency we could also say "Enable" here since we used it already for the chat and so on.
```suggestion
                <span class="ml-2">Enable Moderated Chat</span>
```